Packs/CortexXDR/ReleaseNotes/4_10_17.md

```diff
@@ -0,0 +1,16 @@
+
+#### Incident Types
+
+- **Cortex XDR - XCLOUD Cryptojacking**
+
+#### Layouts
+
+##### Cortex XDR - XCLOUD Cryptojacking
+
+- Changes the name to 'XCLOUD Cryptojacking'
+
+#### Playbooks
+
+##### Cortex XDR - Cloud Cryptojacking
+
+- Fixes a typo in the If-Then-Else transformer
```

Comment:
I saw this fix only on the XSIAM playbook - not in the XSOAR playbook

Reply:
looks good in XSOAR